### ISSUE TITLE
Failing IT on module jqassistant/plugin/testng

### ISSUE THREAD

**Issue by khmarbaise** [buschmais/jqassistant#252]
https://ci.buschmais.com/job/jqassistant-integration/858/ so after checking the following module: jqassistant/plugin/testng is the problem, cause it contains both artifacts junit and testng. I would suggest to simple remote junit dependency directly from the project, cause testng has it transitively.

**Comment by khmarbaise** [comment]
The strange thing is that the following can be found in the log output:

```
Caused by: java.lang.ClassNotFoundException: com.google.inject.Module
	at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.getDeclaredMethods0(Native Method)
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2688)
	at java.lang.Class.getDeclaredMethod(Class.java:2115)
	at org.neo4j.kernel.impl.util.DebugUtil.trackTest(DebugUtil.java:332)
	at org.neo4j.kernel.impl.util.Neo4jJobScheduler.init(Neo4jJobScheduler.java:54)
	at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.init(LifeSupport
```

**Comment by obfischer** [comment]
I will have a look at it.

**Comment by DirkMahler** [comment]
Just a little context: The TestNG dependency only exists for providing a test class that can be scanned. The test framework of jQA itself is based on JUnit. So surefire should only execute the JUnit based test, this has been the reason for specifying the provider (at the wrong location in the parent POM, shoulv have been the module). 
If there's a better solution I'm open to it.

**Comment by obfischer** [comment]
We could exclude this TestNG test class but for the moment it would be enough to specify the JUnit provider manually. We can change this if we move all plugins to separat projects. I will add an comment on this in the POM.

**Comment by obfischer** [comment]
Relates to #246 and #249

**Comment by obfischer** [comment]
I added an comment on the configuration in the POM.

**Comment by obfischer** [comment]
Will be closed #246 contains a correction for this issue.
